docs/argocd-cm.yaml

```diff
@@ -97,6 +97,8 @@ data:
       - repositories.stash.appscode.com
       kinds:
       - Snapshot
+      clusters:
+      - '*'
```

Comment:
Isn't having '*' for clusters the same thing as omitting it entirely? Or are you adding it to round out the documentation?

Reply:
Nope. If you omit, you get no match.

Reply:
OK i think clusters should be additive. This is similar to the behavior esourceNames field with rbac rules. Especially since I don't think the cluster filter is entirely useful, and most people will want the same filters in all clusters.  I'll file an issue to fix this